src/DXTokenRegistryMapping.ts

```diff
@@ -0,0 +1,31 @@
+import { AddToken, RemoveToken } from '../generated/DXTokenRegistry/DXTokenRegistry'
+import { TokenRegistry } from '../generated/schema'
+
+export function handleAddToken(event: AddToken): void {
+  const id = event.params.listId.toHex()
+
+  if(id == '4') {
```

Comment:
Is the result of toHex() prefixed by '0x'? I would have checked via toI32() == 4, assuming listId couldn't get out of bounds. Alternatively, could use toString() here instead.